examples/data-fetch/pages/index.js

```diff
@@ -2,21 +2,25 @@ import React from 'react'
 import Link from 'next/link'
 import fetch from 'isomorphic-unfetch'
 
-function Index(props) {
+function Index({stars}) {
   return (
     <div>
-      <p>Next.js has {props.stars} ⭐️</p>
+      <p>Next.js has {stars} ⭐️</p>
       <Link href="/preact">
         <a>How about preact?</a>
       </Link>
     </div>
   )
 }
 
-Index.getInitialProps = async () => {
+export async function getStaticProps() {
```

Comment:
Isn’t `getServerSideProps` more appropriate in this case? The number of stars changes frequently, so remembering the reading on build will result in stale data.

Reply:
We'll allow revalidation of static pages in the future. The example can be updated again once it's out.